components/Admin/OrderContent.js

```diff
@@ -0,0 +1,115 @@
+import { ExpandMore } from '@mui/icons-material';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Divider,
+  Grid,
+  Stack,
+  Box,
+} from '@mui/material';
+import { PropTypes } from 'prop-types';
+import React from 'react';
+
+import { OrderPropType } from '../../globals';
+const OrderContent = ({ order, isMobile }) => {
+  const date = new Date(order.date);
+  const formattedTime = `${date.getHours().toString().padStart(2, '0')}:${date
+    .getMinutes()
+    .toString()
+    .padStart(2, '0')}`;
+  return (
+    <Accordion sx={{ maxWidth: '750px', minWidth: isMobile ? '95%' : '70%' }}>
```

Comment:
Note that there are some irregularities in width.

Also to note: we have a `dynamic-width` class in `inventory/src/scss/inventoryBase.scss`which u can import. Would be nice to standardise the responsive width. But not sure how it will look like, not pertinent now

<img width="332" alt="Screenshot 2023-07-20 at 10 59 26 PM" src="https://github.com/sjb-deck/deck/assets/55698556/d1626f79-fb43-41db-8489-cdd844450bc5">